tests/test_inverses.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+import pytest
+import torch
+
+import pytorch_finufft
+
+
+def check_t2_ifft_undoes_t1(N: int, dim: int, device: str) -> None:
```

Comment:
it strikes me that maybe we need to name `t1_ifft` to be the one that is actually type 2 ...

Reply:
Yeah, happy to discuss naming. We could also be clearer in the docstring which function this inverts, not just which one it wraps